helm/alfresco-content-services-community/templates/ingress-repository.yaml

```diff
@@ -11,6 +11,12 @@ metadata:
     nginx.ingress.kubernetes.io/session-cookie-hash: "sha1"
     # Default file limit (1m) check, document(s) above this size will throw 413 (Request Entity Too Large) error
     nginx.ingress.kubernetes.io/proxy-body-size: {{ .Values.repository.ingress.maxUploadSize }}
+    nginx.ingress.kubernetes.io/configuration-snippet: |
```

Comment:
I'm not so sure if I'm missing something here. The Search APIs are protected already (see https://github.com/Alfresco/alfresco-search-deployment/commit/f115726e3992e1780aa473b25763ae89780baf86#diff-5a4fd484789954bb21329cd0ca141816). Is this still needed? Can you please talk to @amohammedalfresco about this change please? He added the change for protecting Search APIs. Do you block the access completely with this change? Will anyone be able to access the Solr Admin UI if needed?

Reply:
Having another look at the location definition the Solr Admin UI should not be affected. Abdul's change was for the /solr endpoint and you try to close the internal call. Correct me if I'm wrong please. If that's the case and Abdul can confirm I'm ok with the change. If I misunderstood or missed anything please let me know and we can discuss. Thanks.

Reply:
This location here is the ingress rule for the repository. It defines what parts of `/alfresco` are visible from the outside world.
SOLR pods should talk to the service directly and are not affected by this.
The other commit you are referring to are the bits of `/sor` that are exposed to the outside world.
Btw, IMHO it is a mistake that we have any ingress rules for `/solr` at all. This should be strictly cluster internal and never ever be exposed to the outside world. (Note that admins can always `kubctl proxy` into the cluster)

Reply:
OK, thanks for the clarification, Stefan. Regarding the solr endpoint I can raise a ticket. If we have a way of accessing the Solr Admin UI we should not expose solr endpoint at all then. The password protection can be removed in this case. We will investigate this option.